### DISCUSSION TITLE
What's next for PHEX?

### DISCUSSION THREAD

**Discussion by afkvido** [Prodigy-Hacking/ProdigyMathGameHacking#1269] [Questions] [answered]
# What's next for PHEX?

What I mean is... what are we working on?

Are we finally going to fix Rune Run? 
Are we going to attempt to Bypass 418?
Are we going to make other new hacks?

Are we going to fix all our current <a href="https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/issues">
	<img alt="Issues" src="https://img.shields.io/github/issues-raw/Prodigy-Hacking/ProdigyMathGameHacking">
</a> 

And also.... What's next for PMGH, this community?

**Comment by afkvido** [comment]
I would want to fix our <a href="https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/issues">
	<img alt="Issues" src="https://img.shields.io/github/issues-raw/Prodigy-Hacking/ProdigyMathGameHacking">
</a>

**Reply by donaldli2020** [reply]
👍

**Comment by 00100000** [comment]
A lot of us are busy with school. If you'd like to help, you should contribute through your own pull requests.

**Reply by afkvido** [reply]
you are not a fan of my pull requests

**Reply by 00100000** [reply]
Because they are not bundled and are tiny! I'm a little bit of a hypocrite and haven't contributed much, though, but I'll be spending more time on PMGH soon.

**Comment by PatheticMustan** [comment]
to clarify phex is unrelated to these actual fixes, it just injects the modified game files and disables CORS

**Reply by afkvido** [reply]
i know it does that..... so does that mean rune run will never work?

**Reply by 00100000** [reply]
????? Rune Run bugs don't really have anything to do with PHEX iirc ?????
u sure u know what it does?

**Reply by afkvido** [reply]
Rune run doesn't load with the cheat injection active (not WCM or some items, just having the injection stops rune run)

**Comment by afkvido** [comment, marked as answer]
closing this thread, it's not been too helpful
